extension/test/test-logger.js

```diff
@@ -10,7 +10,13 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
   var message = '';
 
   return function (status) {
-    var line = {}, hasMore;
+    var line = {}, hasMore, isSuccessStatus;
+
+    isSuccessStatus = typeof status == "number";
+    test.assert(isSuccessStatus, status["message"] ? status.message : status);
+    if (!isSuccessStatus) {
+      return;
+    }
```

Comment:
@whimboo, did I answered your question?
I'm asking it, because it would be nice to update the tests to use the `test/assert` module.

<em>__xabolcs__ added a note [16 days ago][3]</em>
> > Isn't `test.assert` an instance of the assertion class? Shouldn't you use a specific test method here?

> Are you asking that the `test` object in these tests are equal to the SDK [`assert`][1] ?
No, they aren't. These `test` objects are [`TestRunner`][2] instances.

> Or I don't understand your question. :-/


[1]: https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/test_assert
[2]: https://github.com/mozilla/addon-sdk/blob/1.17/lib/sdk/deprecated/unit-test.js#L43-L489
[3]: #discussion_r31504407